xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -79,12 +79,7 @@ void CGUIWindowPVRBase::SetSelectedItemPath(bool bRadio, const std::string &path
 
 std::string CGUIWindowPVRBase::GetSelectedItemPath(bool bRadio)
 {
-  if (!m_selectedItemPaths.at(bRadio).empty())
-    return m_selectedItemPaths.at(bRadio);
-  else if (g_PVRManager.IsPlaying())
-    return g_application.CurrentFile();
-
-  return "";
+  return m_selectedItemPaths.at(bRadio);
```

Comment:
As we call `SetSelectedItemPath()` before we tune to a channel, `m_selectedItemPaths will always hold a valid channel path and we don't need to check `g_PVRManager.IsPlaying()` here.